folium/folium.py

```diff
@@ -664,6 +666,31 @@ def fit_bounds(self, bounds, padding_top_left=None,
         self.template_vars.update({'fit_bounds': fit_bounds_str})
 
 
+    def _auto_bounds(self):
+        if 'fit_bounds' in self.template_vars:
```

Comment:
I thought of this too and it'd be cool to have. Since folium is basically built on top of Leaflet, I'd say it's good to follow closely what Leaflet does.

Leaflet allows to fit bounds generically and allows to get bounds from any feature or FeatureGroup. In a future implementation where we'll be able to express featuregroups or layers easily in folium, I'd love to include this, but at the moment I'm for leaving things simple. :smile: